src/core/nativeApi.ts

```diff
@@ -1,8 +1,11 @@
 import { FfiMsg, FfiResult } from './generated'
+import { writeFfiMsg } from './serialization.generated'
+import { Sink } from 'ts-rust-bridge-bincode'
 
-const ref = require('ref')
-const ffi = require('ffi')
-const util = require('util')
+import * as ref from 'ref'
+// const ref = require('ref')
```

Comment:
remove it pls, and maybe move imports from 3rd party libs to the top (3rd party, empty line, our deps, empty line, the rest of the code)

Reply:
what exactly do you want to remove?

Reply:
that commented line :-)